meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mm-lpddr4-evk/lmp.cfg

```diff
@@ -110,3 +110,4 @@ CONFIG_POWER_I2C=y
 # CONFIG_VIDEO_LINK is not set
 # CONFIG_DISTRO_DEFAULTS is not set
 # CONFIG_SHA_HW_ACCEL is not set
+# CONFIG_SET_DFU_ALT_INFO is not set
```

Comment:
We will need a similar change for sec and others, might be better to fix this in u-boot itself.

Reply:
I see what happened, the standard config this is fine because EFI capsule update is also defined, which has a weak definition for the function. I think we should just disable other configs like DFU, let me have a quick look.

Reply:
The problem is this can't be reproduced in the pure u-boot. If CONFIG_SET_DFU_ALT_INFO  is not in defconfig, it stays disabled. But when the recipe u-boot-fio builds, this option turns out enabled. I'll investigate this behavior tomorrow.

Reply:
Happens with our fragment and not in pure u-boot as the EFI options will add a weak function definition and won't cause the problem. The issue is related to the removals we had + additions on the latest bsp.

Which is why I just disabled the new options in #888, no need to spend more time on this I would say.